interop-tests/Cargo.toml

```diff
@@ -26,7 +26,7 @@ libp2p-mplex = { path = "../muxers/mplex" }
 mime_guess = "2.0"
 redis = { version = "0.23.0", default-features = false, features = ["tokio-comp"] }
 rust-embed = "6.7"
-thirtyfour = "0.31"
+thirtyfour = "0.32.0-rc.8" # https://github.com/stevepryde/thirtyfour/issues/169
```

Comment:
I am okay with merging this using a pre-release but please pin it using `=`. Pre-releases are technically semver-compatible but also allow for breaking changes between them.

Reply:
done